frontend/packages/ceph-storage-plugin/src/components/attached-devices-mode/lso-disk-inventory/disk-replacement-modal.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import {
+  ModalTitle,
+  ModalBody,
+  ModalSubmitFooter,
+  createModalLauncher,
+  ModalComponentProps,
+} from '@console/internal/components/factory';
+import { TemplateModel, TemplateInstanceModel, SecretModel } from '@console/internal/models';
+import { CEPH_STORAGE_NAMESPACE } from '../../../constants';
+import {
+  k8sGet,
+  k8sCreate,
+  TemplateKind,
+  TemplateInstanceKind,
+  apiVersionForModel,
+  K8sResourceCommon,
+} from '@console/internal/module/k8s';
```

Comment:
This import needs to be moved before the relative one.

Reply:
done